### PR TITLE
Knative 1.7 release blog post

### PR DESCRIPTION
**Pull request by matzew**
Signed-off-by: Matthias Wessendorf <mwessend@redhat.com>

## Proposed Changes 

- Adding a draft of the 1.7 announce blog post

### CODE REVIEW

File: blog/docs/releases/announcing-knative-v1-7-release.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
+**Authors: [Matthias Wessendorf](https://github.com/matzew) (Red Hat), [Dave Protasowski](https://github.com/dprotaso) (VMware)**
```
Comment (csantanapr) [inline blog/docs/releases/announcing-knative-v1-7-release.md:1]: Add a front matter in the first line like it is in v1.5

Example from 1.5
---
title: "v1.5 release"
linkTitle: "v1.5 release"
Author: "[dprotaso](https://twitter.com/lintinmybelly)"
Author handle: https://github.com/dprotaso
date: 2022-06-06
description: "Knative 1.5 release announcement"
type: "blog"
---

after it also add the authors and date like you already have

Comment (matzew) [reply blog/docs/releases/announcing-knative-v1-7-release.md:1]: @csantanapr I basically copied from 1.6 announcement done by @evankanderson, which also does not have this.

Comment (abrennan89) [reply blog/docs/releases/announcing-knative-v1-7-release.md:1]: @csantanapr I think this was the frontmatter from Hugo days that we no longer use?

Comment (abrennan89) [reply blog/docs/releases/announcing-knative-v1-7-release.md:1]: I do think the author names etc should be under the heading though, for this and the 1.6 post, it looks weird that they're above the page title

**Comment by dprotaso** [comment]
/lgtm
/approve

Let's not hold this up - future edits can be done in another PR

**Pull request merged**